web/src/main/java/org/cbioportal/web/util/StudyViewFilterApplier.java

```diff
@@ -211,22 +219,29 @@ public SampleIdentifier apply(Sample sample) {
             List<String> studyIds = new ArrayList<>();
             List<String> sampleIds = new ArrayList<>();
             studyViewFilterUtil.extractStudyAndSampleIds(sampleIdentifiers, studyIds, sampleIds);
-            List<Integer> ampEntrezGeneIds = copyNumberGeneFilter.getAlterations().stream().filter(a -> 
-                a.getAlteration() == 2).map(CopyNumberGeneFilterElement::getEntrezGeneId).collect(Collectors.toList());
+            
+            List<String> ampGeneIds = copyNumberGeneFilter.getAlterations().stream().filter(a -> 
+            a.getAlteration() == 2).map(CopyNumberGeneFilterElement::getGeneId).collect(Collectors.toList());
```

Comment:
Is this magic number defined in an enum somewhere?

Reply:
Updated